scripts/dispatch-dbt-workflow.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+GH_API_REPO="https://api.github.com/repos/ccao-data/data-architecture"
+GH_DBT_WORKFLOW="test_dbt_models.yaml"
+
+# First auth as a GitHub app using JSON Web Token (JWT).
+# Uses a local PEM file and python lib to construct the JWT
+GH_JWT=$(python3 scripts/get-jwt.py)
```

Comment:
**[Suggestion, non-blocking]** Since most of these commands are piping their output straight into variables, I'm guessing that the commands aren't going to produce any useful logs that could help us debug in case of a failure. I wonder if some simple log statements announcing each step would at least help us narrow down the source of a given problem? Low confidence in this, and we can always add this instrumentation later, so up to you if you want to take this suggestion!

```suggestion
# First auth as a GitHub app using JSON Web Token (JWT).
# Uses a local PEM file and python lib to construct the JWT
echo "Generating JWT to send to GitHub auth"
GH_JWT=$(python3 scripts/get-jwt.py)
```

Reply:
Good call! Added in https://github.com/ccao-data/service-sqoop-iasworld/pull/5/commits/0dc92107abbbc9da2a1de2a1d1d46f0a6b1a85a5.